core/src/main/java/org/kiwix/kiwixmobile/core/main/KiwixWebView.kt

```diff
@@ -180,6 +180,15 @@ open class KiwixWebView @SuppressLint("SetJavaScriptEnabled") constructor(
           Log.w("kiwix", "Couldn't save image", e)
           instance.toast(R.string.save_media_error)
         }
+
+        MediaScannerConnection.scanFile(
```

Comment:
Add a comment explaining what is happening here.

Reply:
I have added a new commit can you please take a look.